tests/lib/wast/src/wasi_wast.rs

```diff
@@ -21,6 +21,7 @@ pub struct WasiTest<'a> {
     mapped_dirs: Vec<(&'a str, &'a str)>,
     temp_dirs: Vec<&'a str>,
     assert_return: Option<AssertReturn>,
+    provide_stdin: Option<ProvideStdin<'a>>,
```

Comment:
I will name it `stdin` rather than `provide_stdin` (what can be stdin otherwise?)

Reply:
Fair point!

My thought was just making it symmetrical with `assert_stdout`, I can change it to `stdin` though